src/test/resources/mgmt/user_refresh_tokens.json

```diff
@@ -0,0 +1,44 @@
+{
+  "tokens": [
+    {
+      "id": "tokenId1",
+      "user_id": "userId1",
+      "created_at": "2024-06-26T09:10:26.643Z",
+      "updated_at": "2024-06-26T09:10:27.131Z",
+      "expires_at": "2024-07-03T09:10:26.643Z",
+      "client_id": "clientId1",
+      "session_id": "sessionId1",
+      "rotating": false,
+      "resource_servers": [
+        {
+          "audience": "https://api.example.com",
+          "scopes": [
+            "read:examples",
+            "write:examples"
+          ]
+        }
+      ]
+    },
+    {
+      "id": "tokenId2",
+      "user_id": "userId1",
+      "created_at": "2024-06-26T09:10:26.643Z",
+      "updated_at": "2024-06-26T09:10:27.131Z",
+      "expires_at": "2024-07-03T09:10:26.643Z",
+      "client_id": "clientId2",
+      "session_id": "sessionId2",
+      "rotating": true,
+      "resource_servers": [
+        {
+          "audience": "https://api.example.com",
+          "scopes": [
+            "read:examples",
+            "write:examples"
+          ]
+        }
+      ]
+    }
+  ],
+  "next": "token1",
+  "total": 11
```

Comment:
I think we are missing `device` here for testing purposes?

Reply:
Yes, it was not added earlier not sure about it. I have added device for testing